gpii/node_modules/canopyMatchMaker/test/web/html/CanopyMatchMakerUtilitiesTests.html

```diff
@@ -4,21 +4,27 @@
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
         <title>GPII Canopy MatchMaker tests</title>
 
-        <link rel="stylesheet" media="screen" href="../../../../../../node_modules/infusion/tests/lib/qunit/css/qunit.css" />
+        <link rel="stylesheet" media="screen" href="/node_modules/infusion/tests/lib/qunit/css/qunit.css" />
```

Comment:
Is this change to use absolute paths rather than continuing with relative paths something that must happen for using testem? This change would not allow developers to run an individual browser test within a local browser using the own web server.

To provide more detail of my use case, I always have MAMP running on my machine to serve as a local web server. When writing a browser unit test, I usually open that file in my local browser using a url like `http://localhost/gpii/node_modules/universal/gpii/node_modules/canopyMatchMaker/test/web/html/CanopyMatchMakerUtilitiesTests.html` for debugging. This path change causes the load of the test file with this approach throws many `404 Not Found` error. Is this method not supported with the switch to use testem?